framework/src/application/application.ts

```diff
@@ -199,7 +201,7 @@ export class Application {
 		this._plugins[pluginAlias] = pluginKlass as InstantiablePlugin<BasePlugin>;
 	}
 
-	public overrideModuleOptions(alias: string, options?: object): void {
+	public overridePluginOptions(alias: string, options?: object): void {
```

Comment:
after defining the above type, u could use the type instead of `object`?

Reply:
We are not sure what configuration we get fro any plugin.